### PR TITLE
add scripts folder & add first script

### PR DESCRIPTION
**Pull request by omasopust**

### CODE REVIEW

File: scripts/ondrej.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// ajskdfjlsadjflkasjdfkljasf
+
+// muj skript pico GTFO
```
Comment (JiriPohanka) [inline scripts/ondrej.js:3]: I dont think this kind of language is appropriate in a public repo

File: scripts/ondrej.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
```
Comment (HoraRadosti) [inline scripts/ondrej.js:4]: pico+-

File: scripts/ondrej.js (hunk 1 above)
Comment (omasopust) [reply scripts/ondrej.js:3]: yes

File: scripts/ondrej.js (hunk 2 above)
Comment (omasopust) [reply scripts/ondrej.js:4]: spíš mínus

File: scripts/ondrej.js (hunk 1 above)
Comment (JiriPohanka) [reply scripts/ondrej.js:3]: LGTF

**Review by JiriPohanka — APPROVED** [review]

**Pull request merged**